spec/services/open_library_service_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe OpenLibraryService do
+  response_path = './spec/fixtures/stubs_and_mocks/open_library_service_response_body.txt'
+
+  before do
+    stub_request(:any, %r{.*\/api\/books\?bibkeys=ISBN:.*&format=json&jscmd=data}) \
+      .to_return(body: File.open(response_path))
+  end
+
+  describe '#book_info' do
+    context 'With existent isbn' do
+      existent_isbn = '0385472579'
```

Comment:
Always use [`let`s](https://relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let) to store values! Never use a simple assignation!